spec/generators/rspec/install/install_generator_spec.rb

```diff
@@ -27,7 +27,7 @@ def require_rspec_rails
     match(/^require 'rspec\/rails'$/m)
   end
 
-  def use_transactional_fixtures
+  def have_transactional_fixtures
```

Comment:
How do you feel about `set_transactional_fixtures` or `enable_transactional_fixtures`?